src/zerocoin_v3.cpp

```diff
@@ -107,8 +107,10 @@ bool CheckSpendZcoinTransactionV3(
 		int nHeight,
 		bool isCheckWallet,
 		CZerocoinTxInfoV3 *zerocoinTxInfoV3) {
+	int txHeight = chainActive.Height();
```

Comment:
We need to lock `cs_main` before access `chainActive`.

Reply:
hmm, I got this after merge. but seem like it never be used.